Mage/Module.cs

```diff
@@ -90,6 +90,11 @@ public class Module<T> : Singleton<T> where T : class, new() {
 	
 	public void setupUsercommands (Action<Exception> cb) {
 		logger.info ("Setting up usercommands");
+
+		if (commands == null) {
```

Comment:
OH right, yeah i got it, you have a point, ill initialise the list above the return. This would avoid a NULL reference exception if they try to use a command. Instead it will spit out element doesn't exist.